### DISCUSSION TITLE
Error firebase_app_distribution get-latest-build-version

### DISCUSSION THREAD

**Discussion by BeesUser** [codemagic-ci-cd/codemagic-docs#2239] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

begin build and error in 
Post-clone script 

### Expected results

Links to Firebase links

### Actual results

Failed to communicate with Google. Request contains an invalid argument.

### Build id (optional)

656bcfd7e8923d52b6b68761

**Comment by himesh-cm** [comment]
Hi @KropVOo, the issue seems to be with `FIREBASE_CREDENTIALS`. Can you please check if the value is properly added in your post-build script?
Please try it locally first to see if it works as expected.

**Reply by BeesUser** [reply]
i found error, please change documentation, need use project number, not project ID.
Can you upgrade this CLI tools and return not only build version, but and links to this version on FIREBASE?

**Reply by himesh-cm** [reply]
Hi @KropVOo, thanks for the feedback. I will check the documentation. 
You can follow the issue #2258.

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!

**Comment by MehmetKaranlik** [comment]
Documentation is still wrong it says `project ID` while correct one should be `project number`. Big thanks @KropVOo
